src/entites/user.entity.ts

```diff
@@ -1,10 +1,11 @@
+import { Guid } from "guid-typescript";
 import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
 
 @Entity()
 export class User {
     
     @PrimaryGeneratedColumn()
-    id:number;
+    id:Guid;
```

Comment:
hier brauchen wir keine Fremd-Library. TypeORM kann das von Hause aus: https://stackoverflow.com/questions/47651513/typeorm-auto-generated-uuid-in-postgresql